packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -1374,11 +1374,11 @@ - (NSDictionary *)multiFactorResolverToDict:(FIRMultiFactorResolver *)resolver {
 }
 
 - (NSString *)getJSFactorId:(NSString *)factorId {
-  if ([factorId isEqualToString:@"1"]) {
+  if (factorId == nil || [factorId isEqualToString:@"1"]) {
     // Only phone is supported by the front-end so far
     return @"phone";
   }
-
+    
```

Comment:
This is a really minor point, but please don't mix spacing changes into functional changes as it makes the diff unnecessarily larger. Additionally our formatting is all automated (via `yarn lint:ios:fix` if I remember correctly, and checked via `yarn lint:ios:check` again if I remember correctly - so this will likely cause a CI failure

Spacing changes just generally best avoided...

```suggestion

```